tables/rubin_early_dr_scenario.tex

```diff
@@ -40,9 +40,15 @@
 %
 DRP Difference Images and DIA Catalogs      &  --  &   \mycirc[RubinDarkTeal]  &    --  &    \mycirc[RubinGray1]  &    \mycirc[RubinDarkTeal]  &    \mycirc[RubinDarkTeal]  &  \mycirc[RubinDarkTeal]  &    \mycirc[RubinDarkTeal]  \\  \hline
 %
-DRP SSP Catalogs   &   --  &   -- &    \mycirc[RubinDarkTeal]    &     --  &    \mycirc[RubinGray1]  &    \mycirc[RubinDarkTeal]  &  \mycirc[RubinDarkTeal]  &    \mycirc[RubinDarkTeal]  \\  
+DRP SSObject Catalog   &   --  &   -- &    \mycirc[RubinDarkTeal]    &      \mycirc[RubinDarkTeal]    &    \mycirc[RubinDarkTeal]  &    \mycirc[RubinDarkTeal]  &  \mycirc[RubinDarkTeal]  &    \mycirc[RubinDarkTeal]  \\  \hline
```

Comment:
Propose to revert this one back to just a single row like it was before, as we discussed.